frontend/src/pages/admin/Template/sections/Environment.vue

```diff
@@ -88,8 +91,8 @@ import FormRow from '@/components/FormRow'
 import FormHeading from '@/components/FormHeading'
 import LockSetting from '../components/LockSetting'
 import ChangeIndicator from '../components/ChangeIndicator'
-import { TrashIcon, PlusSmIcon, LockClosedIcon } from '@heroicons/vue/outline'
-
+import { TrashIcon, PlusSmIcon, LockClosedIcon, CogIcon } from '@heroicons/vue/outline'
+const RESERVED_ENV = ['FF_PROJECT_ID', 'FF_PROJECT_NAME', 'FF_DEVICE_ID', 'FF_DEVICE_NAME', 'FF_DEVICE_TYPE', 'FF_SNAPSHOT_ID', 'FF_SNAPSHOT_NAME']
```

Comment:
How about we consider `FF_*` to be reserved rather than try to maintain a list in multiple places?

Reply:
ok, can do. 

will push change in a moment

Reply:
oh - wait. Does that mean at the backend too (not just front end)

Reply:
If we say `FF_*` are reserved, then the API should not allow a user to set any env vars starting `FF_*`.

Reply:
resolved in [2dd72bc](https://github.com/flowforge/flowforge/pull/1039/commits/2dd72bceaab1691037fa8ba2d82c38d63da4d0d3)